tasks/db_tasks.py

```diff
@@ -107,13 +111,13 @@ def seed_dev_db(c, fundround=None, appcount=None):
                         "Please type the fund-round to seed:"
                         f"fund-rounds available to seed: "
                         f"{new_line} - {f' {new_line} - '.join(config.keys())}"
+                        f"{new_line} > "
                     ),
                 )
-                fund_round_input = str(
-                    _echo_input("Please type the " "fund-round to seed:")
-                )
```

Comment:
Are we purposefully getting rid of this?

Reply:
yeah it wasnt coming our right in the terminal, ive hopefully improved the seeding prompts now